adapters/src/main/java/io/realm/RealmRecyclerViewAdapter.java

```diff
@@ -33,20 +33,27 @@
  * The RealmAdapter will stop receiving updates if the Realm instance providing the {@link OrderedRealmCollection} is
  * closed.
  *
- * @param <T> type of {@link RealmModel} stored in the adapter.
+ * @param <T>  type of {@link RealmModel} stored in the adapter.
```

Comment:
There is a guideline for this: https://github.com/realm/realm-android-adapters/blob/master/CONTRIBUTING.md#javadoc and I think being consistent is important. :)